test/inductor/test_flex_decoding.py

```diff
@@ -825,6 +829,57 @@ def test_builtin_score_mods_different_block_size(
         )
         self.run_test(score_mod, dtype, block_mask=block_mask, device=device)
 
+    @common_utils.parametrize("dtype", test_dtypes_fast)
+    @common_utils.parametrize("use_tma", test_tma_options)
+    def test_tma_decoding(self, device, dtype: torch.dtype, use_tma):
```

Comment:
nit just skip the test if device does not support tma, no need to parametrize if there is 1 option

Reply:
The newly added test case will be skipped if `has_triton_tma_device` is `False`, and the `False` scenario should have been covered by other test cases.